central-service/controllers/problem.go

```diff
@@ -0,0 +1,95 @@
+package controllers
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"strconv"
+
+	"github.com/acmutd/bsg/central-service/models"
+	"github.com/acmutd/bsg/central-service/services"
+	"github.com/labstack/echo/v4"
+)
+
+type ProblemController struct {
+	problemService *services.ProblemService
+}
+
+func InitializeProblemController(service *services.ProblemService) ProblemController {
+	return ProblemController{service}
+}
+
+// An endpoint containing logic used to create a new problem based on provided parameters
+func (controller *ProblemController) CreateNewProblemEndpoint(c echo.Context) error {
```

Comment:
What is the use case for this endpoint? Are we using this endpoint to manually add a leetcode problem into our database?

Reply:
Good point, we don't actually need this to add problems to our DB. 
We also don't need the update problems endpoint for the same reason, so I'll remove that as well.

And I remembered we also want a fetch all problems endpoint so I'll work on that also.

TODO for me:
- [ ] Remove Create endpoint
- [ ] Remove Update endpoint
- [ ] Add fetch all endpoint (with pagination)